ApiDoctor.Validation/DocSet.cs

```diff
@@ -659,15 +658,18 @@ private void ReadDocumentationHierarchy(string path)
                 throw new FileNotFoundException(string.Format("Cannot find documentation. Directory doesn't exist: {0}", sourceFolder.FullName));
             }
 
-            var markdownFileInfos = sourceFolder.GetFiles(DocumentationFileExtension, SearchOption.AllDirectories);
-            var markdownFiles = markdownFileInfos.
-                Where(fi => foldersToSkip.All(folderToSkip => !fi.DirectoryName.Contains(folderToSkip)) && !filesToSkip.Contains(fi.Name)).
-                Select(fi => new DocFile(this.SourceFolderPath, this.RelativePathToFile(fi.FullName), this) { WriteFixesBackToDisk = this.writeFixesBackToDisk });
+            var markdownFileInfos = sourceFolder.GetFiles(DocumentationFileExtension, SearchOption.AllDirectories)
+                .Where(x => !x.Name.Contains("intune")).ToArray();
```

Comment:
is this necessary? what in intune isn't passing? if we must add this, let's not put it in the code. instead, let's add 'pathSubstringsToSkip' (or something) to one of the config files (like schema-config, even though that's not super appropriate) and read it from there. that way only the graph repo will have this configuration

Reply:
Good point. Yes intune has a lot of errors, that I cannot fix ( I was fixing the docs manually earlier on). I was informed intune docs are autogenerated by another system.